goon/browserassets/js/browserOutput.js

```diff
@@ -171,25 +171,67 @@ function setHighlightColor(match) {
 	match.style.background = opts.highlightColor
 }
 
+function escapeRegex(input){ // put this in a function incase it ever needs to be used elsewhere, makes code cleaner
+	return input.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+}
+
 //Highlights words based on user settings
 function highlightTerms(el) {
 	if(regexHasError) return; //just stop right there ig the regex is gonna except
+	function highlightRecursor(element,term){ //recursor function to do the highlighting proper
+		var regex = new RegExp("("+term+")","gi")
+		function replace(str) {
+			return str.replace(regex,'<span style="background-color:'+opts.highlightColor+'">$&</span>')
+		}
+		var s=''
```

Comment:
Style: rest of file has a space on both sides of assignment (same below)

Reply:
reeeeeeeee

Reply:
Isn't this missing a semicolon? I'd expect this to fail to run

Reply:
Javascript doesnt particularly care about colons, unless theres a case in where the separation between two statements can be ambiguous, ex:
```
var a = "a"
var b = "b"
```
is okay, but it will throw an error if you do:
```
var a = "a" var b = "b"
```
unless theres a semicolon between the two statements.